src/components/map/RoutePoiButton.scss

```
@@ -0,0 +1,37 @@
+.route__poi-button {
+  align-items: center;
+  appearance: none;
+  background-color: #3EB2FD;
+  background-image: linear-gradient(1deg, #4F58FD, #149BF3 99%);
+  background-size: calc(100% + 20px) calc(100% + 20px);
+  border-radius: 100px;
+  border-width: 0;
+  box-shadow: none;
+  box-sizing: border-box;
+  color: #FFFFFF;
+  cursor: pointer;
+  display: inline-flex;
+  font-size: 0.75rem;
+  height: auto;
+  justify-content: center;
+  line-height: 1.5;
+  padding: 0.2rem 0.4rem;
+  position: absolute;
+  text-align: center;
+  text-decoration: none;
+  transition: color .2s ease-in-out, box-shadow .2s ease-in-out;
+  user-select: none;
+  -webkit-user-select: none;
+  touch-action: manipulation;
+  vertical-align: top;
+  white-space: nowrap;
+}
+
+.route__poi-button:hover {
+  box-shadow: inset 100px 0 0 0 #20afe3;
+  color: white;
+}
+
+.route__poi-button:focus:not(:active) {
+  box-shadow: rgba(40, 170, 255, 0.25) 0 0 0 .125em;
+}
```

Thanks to SCSS, these can be nested:
```suggestion

  &:hover {
    box-shadow: inset 100px 0 0 0 #20afe3;
    color: white;
  }

  &:focus:not(:active) {
    box-shadow: rgba(40, 170, 255, 0.25) 0 0 0 .125em;
  }
}
```